python_modules/libraries/dagster-aws/dagster_aws/s3/io_manager.py

```diff
@@ -149,9 +149,16 @@ def handle_output(self, context: OutputContext, obj: Any) -> None:
         return self.inner_io_manager().handle_output(context, obj)
 
 
+# S3PickleIOManager used to be named ConfigurablePickledObjectS3IOManager, keep this symbol around for backcompat
+ConfigurablePickledObjectS3IOManager = S3PickleIOManager
+ConfigurablePickledObjectS3IOManager.__doc__ = """
```

Comment:
thinking something like this for the docstring rather than inheriting from  S3PickleIOManager